lib/api/controllers/routerController.js

```diff
@@ -142,68 +143,207 @@ module.exports = function RouterController(kuzzle) {
-        {verb: 'post', url: '/users/_create', controller: 'security', action: 'createUser'},
-        {verb: 'post', url: '/users/:id', controller: 'security', action: 'updateUser'},
-        {verb: 'post', url: '/_login', controller: 'auth', action: 'login'},
-        {verb: 'post', url: '/_checkToken', controller: 'auth', action: 'checkToken'},
-        {verb: 'post', url: '/_login/:strategy', controller: 'auth', action: 'login'},
-        {verb: 'post', url: '/:index/_bulk', controller: 'bulk', action: 'import'},
-        {verb: 'post', url: '/:index/_refresh', controller: 'admin', action: 'refreshIndex'},
-        {verb: 'post', url: '/:index/:collection/_bulk', controller: 'bulk', action: 'import'},
-        {verb: 'post', url: '/:index/:collection/_search', controller: 'read', action: 'search'},
-        {verb: 'post', url: '/:index/:collection/_count', controller: 'read', action: 'count'},
-        {verb: 'post', url: '/:index/:collection/_create', controller: 'write', action: 'create'},
-        {verb: 'post', url: '/:index/:collection', controller: 'write', action: 'publish'},
-
-        {verb: 'delete', url: '/_deleteIndexes', controller: 'admin', action: 'deleteIndexes'},
-        {verb: 'delete', url: '/roles/:id', controller: 'security', action: 'deleteRole'},
-        {verb: 'delete', url: '/profiles/:id', controller: 'security', action: 'deleteProfile'},
-        {verb: 'delete', url: '/users/:id', controller: 'security', action: 'deleteUser'},
-        {verb: 'delete', url: '/:index', controller: 'admin', action: 'deleteIndex'},
-        {verb: 'delete', url: '/:index/:collection/_query', controller: 'write', action: 'deleteByQuery'},
-        {verb: 'delete', url: '/:index/:collection/_truncate', controller: 'admin', action: 'truncateCollection'},
-        {verb: 'delete', url: '/:index/:collection/:id', controller: 'write', action: 'delete'},
-
-        {verb: 'put', url: '/roles/:id', controller: 'security', action: 'createOrReplaceRole'},
-        {verb: 'put', url: '/roles/:id/_createOrReplace', controller: 'security', action: 'createOrReplaceRole'},
-        {verb: 'put', url: '/profiles/:id', controller: 'security', action: 'createOrReplaceProfile'},
-        {verb: 'put', url: '/profiles/:id/_createOrReplace', controller: 'security', action: 'createOrReplaceProfile'},
-        {verb: 'put', url: '/users/:id', controller: 'security', action: 'createOrReplaceUser'},
-        {verb: 'put', url: '/:index', controller: 'admin', action: 'createIndex'},
-        {verb: 'put', url: '/:index/:collection', controller: 'write', action: 'createCollection'},
-        {verb: 'put', url: '/:index/:collection/_mapping', controller: 'admin', action: 'updateMapping'},
-        {verb: 'put', url: '/:index/:collection/:id/_:action', controller: 'write'},
-        {verb: 'put', url: '/:index/:collection/:id', controller: 'write', action: 'createOrReplace'}
-      ];
+      coverage;
+
+    routes = [
```

Comment:
I'm starting to believe that we should move these routes into a configuration file :-)